### PR TITLE
Update Cloud IDE IP address

### PR DESCRIPTION
**Pull request by jlaneve**
Cloud IDE requests now come from a new IP address. This PR changes the IP to reflect the new one!

Please merge this PR when it's approved.

### CODE REVIEW

File: astro/cloud-ide/security.md [hunk 1]
```diff
@@ -25,8 +25,7 @@ The Cloud IDE stores Python cell outputs in an encrypted S3 bucket. SQL cell out
 
 Astro Cloud IDE requests can come from the following IP addresses:
```
Comment (jwitz) [inline astro/cloud-ide/security.md:26]:
```suggestion
Astro Cloud IDE requests can come from the following IP address:
```

File: astro/cloud-ide/security.md [hunk 2]
```diff
@@ -25,8 +25,7 @@ The Cloud IDE stores Python cell outputs in an encrypted S3 bucket. SQL cell out
...
 
-- `20.237.27.58`
-- `40.76.151.37`
+- `20.232.242.54`
 
 Add these IP addresses to your allowlist to run the Astro Cloud IDE in a private network. These IP addresses are subject to change.
```
Comment (jwitz) [inline astro/cloud-ide/security.md:30]:
```suggestion
Add this IP address to your allowlist to run the Astro Cloud IDE in a private network. This address is subject to change. 
```

**Review by jwitz — APPROVED** [review]
Thanks @jlaneve , Can you confirm whether we still need the "Subject to change" line? After I get confirmation there I'll merge!

**Comment by jlaneve** [comment]
Yes, still subject to change!

**Pull request merged**